08-communicating-results.Rmd

```diff
@@ -1,14 +1,16 @@
 # Communicating Results {#c08-communicating-results}
 
-One of the most important aspects of data analysis is communicating the results to others. This could include other researchers familiar with our survey data or others who will be seeing this data and results for the first time. Ensuring that we are accurately discussing the methodology, analysis, and displaying results is crucial to making sure our audience comprehends what the results are saying. There have been jokes about how we can lie with statistics, so it is our responsibility to discuss and present the results carefully.
+<!-- author review -->
+
+One of the most important aspects of data analysis is communicating the results to others. This could include other researchers familiar with our survey data or others who will be seeing this data and results for the first time. Ensuring that we are accurately discussing the methodology, analysis, and displaying results is crucial to making sure our audience comprehends what the results are saying. There have been jokes about how we can lie with statistics, so it is our responsibility to discuss and present the results carefully.  
 
```

Comment:
We talk about audience later on, but I think it deserves its own section. How does an analyst identify the right audience for communicating results? How do we think about presenting differently based on knowledge and expertise? How do we tailor the approach based on their characteristics?